ArduCopter/GCS_Mavlink.cpp

```diff
@@ -494,12 +494,23 @@ AP_GROUPEND
 
 static const ap_message STREAM_RAW_SENSORS_msgs[] = {
     MSG_RAW_IMU,
+#if INS_MAX_INSTANCES > 1
     MSG_SCALED_IMU2,
+#if INS_MAX_INSTANCES > 2
     MSG_SCALED_IMU3,
+#endif
+#endif
+#if AP_BARO_ENABLED
```

Comment:
We send airspeed data in SCALED_PRESSURE etc.  This change would break that if you didn't have a baro.

Reply:
There is a WHOOP quadcopter that does not have a barometric sensor.
This is the type of copter that was envisioned.
Determine the number of barometric sensors at 0 and verify that SCALED_PRESSURE is being sent.

Reply:
You would have to make this based upon both BARO_MAX_INSTANCES and AIRSPEED_MAX_SENSORS, INS is a complete non fit here.